plugin/kubernetes/setup.go

```diff
@@ -121,7 +121,17 @@ func ParseStanza(c *caddy.Controller) (*Kubernetes, error) {
 	zones := c.RemainingArgs()
 
 	if len(zones) != 0 {
-		k8s.Zones = zones
+		var classfulZones []string
+		for i := 0; i < len(zones); i++ {
+			tmp, err := plugin.ClassfulFromCIDR(zones[i])
+			if err == nil {
+				classfulZones = append(classfulZones, tmp...)
+
+			} else {
+				classfulZones = append(classfulZones, zones[i])
+			}
+		}
+		k8s.Zones = classfulZones
```

Comment:
if this requires plugin changes it's a no-go. If we can't do it on the server level than the whole 'reverse-gen' feature will be removed. As in: nice idea, didn't work too bad

Reply:
No problem to implement it on the server